Example/IGListKitExamples/SectionControllers/DemoSectionController.swift

```diff
@@ -53,7 +56,12 @@ class DemoSectionController: IGListSectionController, IGListSectionType {
     }
 
     func didSelectItem(at index: Int) {
-        if let controller = object?.controllerClass.init() {
+        if let identifier = object?.controllerIdentifier {
+            let storyboard = UIStoryboard(name: "Demo", bundle: nil)
```

Comment:
Noting to myself: this means that any demo using a storyboard will have to put their VCs in the `Demo.storyboard` file. I think that's fine for now.